predator/asp/greedy-target-seed-solving_SA.lp

```diff
@@ -46,32 +39,23 @@ present(M) :- any_seed(M).
 % % a metabolite is present or not
 % { present(M) } :- metabolite(M).
 
-% % products of a feasible reaction are producible
-% producible(M) :- feasible(R), product(M,R).
+% products of a feasible reaction are producible
+producible(M) :- feasible(R), product(M,R).
```

Comment:
The predicate producible is later used to discard solutions where metabolites are chosen present which are neither seeds nor producible by a feasible reaction.